runtime/default.policy

```diff
@@ -362,10 +377,16 @@ rules:
     expression: exec.envs in [~"LD_PRELOAD=*/tmp/*" ,~"LD_PRELOAD=/dev/shm/*" ]
   - id: net_util
     description: A network utility was executed
-    expression: "(exec.comm in NET_UTILS ||\n exec.comm in HTTP_UTILS) &&\ncontainer.id == \"\" && (exec.args not in [ ~\"*localhost*\", ~\"*127.0.0.1*\", ~\"*motd.ubuntu.com*\" ] || process.ancestors.comm != \"50-motd-news\") "
+    expression: |-
+      (exec.comm in NET_UTILS ||
+       exec.comm in HTTP_UTILS) &&
+      container.id == "" && exec.args not in [ ~"*localhost*", ~"*127.0.0.1*", ~"*motd.ubuntu.com*" ]
   - id: net_util_in_container
     description: A network utility was executed in a container
-    expression: "(exec.comm in NET_UTILS ||\n exec.comm in HTTP_UTILS) &&\ncontainer.id != \"\" && (exec.args not in [ ~\"*localhost*\", ~\"*127.0.0.1*\", ~\"*motd.ubuntu.com*\" ] || process.ancestors.comm != \"50-motd-news\") "
+    expression: |-
+      (exec.comm in NET_UTILS ||
+       exec.comm in HTTP_UTILS) &&
+      container.id != "" && exec.args not in [ ~"*localhost*", ~"*127.0.0.1*", ~"*motd.ubuntu.com*" ]
```

Comment:
Can you replace `exec.comm` by `exec.file.name` in these two rules? Otherwise LGTM!

Reply:
`exec.argv0` can also easily be changed to bypass the rule, so I guess we can stick to using `exec.comm` for now until we have better support for busybox.